src/modules/article/article.service.ts

```diff
@@ -497,6 +495,39 @@ export class ArticleService {
     };
   }
 
+  async getArticleHighInteractionStatistics(
```

Comment:
```suggestion
  async getStats(
```
có thể đặt tên như này là đủ.

với những logic thống kê phức tạp hơn, việc cố gắng phản ánh hết vào tên method sẽ khó khăn hơn.
ngoài ra khi spec thay đổi, tên cũ ko phù hợp thì cần sửa lại. Việc này có thể tránh bằng cách chọn 1 cái tên generic hơn ngay từ đầu